<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout" rowstretch="1,6,2,1,0">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>camera id:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QGraphicsView" name="imageView"/>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="stop">
        <property name="text">
         <string>stop</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pause">
        <property name="text">
         <string>pause</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="start">
        <property name="text">
         <string>start</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="open">
        <property name="text">
         <string>open</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="id"/>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QSlider" name="horizontalSlider">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>99</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1070</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="file">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="settings"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>发送模式</string>
    </property>
    <addaction name="actionBFM"/>
    <addaction name="actionG8M"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>接收端</string>
    </property>
    <addaction name="actionLocal"/>
    <addaction name="actionUnity"/>
   </widget>
   <addaction name="file"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="settings">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/iconImage/img/settings.png</normaloff>:/iconImage/img/settings.png</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="actionBFM">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>BFM</string>
   </property>
  </action>
  <action name="actionG8M">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>G8M</string>
   </property>
  </action>
  <action name="actionLocal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>本地</string>
   </property>
  </action>
  <action name="actionUnity">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>unity</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
